cache/image_cache.go

```diff
@@ -45,6 +45,7 @@ func NewImageCacheFromName(name string, keychain authn.Keychain) (*ImageCache, e
 		keychain,
 		remote.WithPreviousImage(name),
 		remote.WithDefaultPlatform(imgutil.Platform{OS: runtime.GOOS}),
+		remote.AddEmptyLayerOnSave(),
```

Comment:
Although... TBH... IDK if that's possible with our fake.

Reply:
I didn't check how to add the test yet, but I will do it

Reply:
Well, I checked and the method `NewImageCacheFromName` is not being covered by test in the image_cache_test.go file, I supposed because it requires access to a registry and complicate the test

Reply:
@natalieparellano I added the acceptance test. Thanks for your help to make it work!